index.js

```diff
@@ -26,22 +26,42 @@ module.exports = (ms, options = {}) => {
 	};
 
 	const parsed = parseMs(ms);
+	// Manually add µs and ns to parsed, could be a pr to parse-ms
```

Comment:
These lines could be removed if https://github.com/sindresorhus/parse-ms/pull/5 gets merged